compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -341,8 +342,25 @@ private void visitFunction(BLangFunction funcNode) {
             return;
         }
         boolean isNilableReturn = funcNode.symbol.type.getReturnType().isNullable();
+
+        BLangType returnTypeNode = funcNode.returnTypeNode;
+        if (returnTypeNode.type.tag == TypeTags.ARRAY && returnTypeNode instanceof BLangArrayType) {
+            analyzeArrayElemImplicitInitialValue(((BLangArrayType) returnTypeNode),
+                    returnTypeNode.pos);
```

Comment:
You mean inline `returnTypeNode`?

Reply:
Yes, AFAIS, it will not exceed 120 character limit.

Eg - `analyzeArrayElemImplicitInitialValue(((BLangArrayType) returnTypeNode), returnTypeNode.pos);`

Reply:
Oh my bad, I didn't see that second argument is on a separate line (Y)